endpoints/_includes/functions.php

```diff
@@ -746,6 +748,30 @@ function helplineSearch($latitude, $longitude)
     return json_decode(get($search_url));
 }
 
+function getBMLTserver($latitude, $longitude, $search_state)
+{
+	$helpline_search_radius=setting('helpline_search_radius');
+	if ($search_state=="1") {
+		if (setting('tomato_helpline_fallback')) {
+			$bmlt_search_endpoint=setting('tomato_url')."/client_interface/json/?switcher=GetSearchResults&data_field_key=longitude,latitude,root_server_uri&sort_results_by_distance=1&long_val={LONGITUDE}&lat_val={LATITUDE}&geo_width=" . $helpline_search_radius;
```

Comment:
since you are only looking for `root_server_uri`, limit to just returning that field to keep performance optimal.